mkt/api/base.py

```diff
@@ -172,6 +172,16 @@ def get_region_from_request(self, request):
         """
         return get_region_from_request(request)
 
+    def get_serializer_class(self):
+        """
+        Return a serializer for the requested API version. Versioned
+        serializers are registered on the default serializer with their version
+        number.
+        """
+        api_version = 'V{v_int}'.format(v_int=self.request.API_VERSION)
```

Comment:
Technically it's just an attribute :) It's not very different from, say, `_meta` attribute on models.